content/reference/admin/private_locations/configuration/azure/index.md

```diff
@@ -68,6 +72,12 @@ control-plane {
       # Use "subnets.name" as returned by Azure CLI:
       # az network vnet subnet list --resource-group MyResourceGroup --vnet-name MyVNet
       subnet-name = "default"
+      
+      # Overwrite JAVA_HOME definition (optional)
+      # java-home = "/usr/lib/jvm/zulu17"
+      
+      # JVM Options (optional)
+      # jvm-options = ["-Xmx6G"]
```

Comment:
Should we split the orchestrator part (how to launch an instance) and the program part (how to configure the simulation and the program)?